www/src/templates/template-api-markdown.js

```diff
@@ -0,0 +1,270 @@
+import normalizeGatsbyApiCall from "../utils/normalize-gatsby-api-call"
+import { sortBy } from "lodash-es"
+
+import Layout from "../components/layout"
+import {
+  itemListDocs,
+  itemListTutorial,
+  itemListContributing,
+} from "../utils/sidebar/item-list"
+import MarkdownPageFooter from "../components/markdown-page-footer"
+import DocSearchContent from "../components/docsearch-content"
+import TableOfContents from "../components/docs-table-of-contents"
+import FooterLinks from "../components/shared/footer-links"
+import Breadcrumb from "../components/docs-breadcrumb"
+import Container from "../components/container"
+import PrevAndNext from "../components/prev-and-next"
+
+const containerStyles = {
+  // we need to account for <Container>'s horizontal padding of
+  // `space[6]` each (1.5rem), plus add a fluffy `space[9]`
+  // of whitespace in between main content and TOC
+  //
+  // could be much cleaner/clearer, please feel free to improve 🙏
+  maxWidth: t =>
+    `calc(${t.sizes.mainContentWidth.withSidebar} + ${t.sizes.tocWidth} + ${
+      t.space[9]
+    } + ${t.space[9]} + ${t.space[9]})`,
+  px: 9,
+}
+
+const getDocsData = location => {
+  const [urlSegment] = location.pathname.split(`/`).slice(1)
+  const itemListLookup = {
+    docs: itemListDocs,
+    contributing: itemListContributing,
+    tutorial: itemListTutorial,
+  }
+
+  return [urlSegment, itemListLookup[urlSegment]]
+}
+
+const mergeFunctions = (data, context) => {
+  const normalized = normalizeGatsbyApiCall(data.nodeAPIs.group)
+
+  const docs = data.jsdoc.nodes.reduce((acc, node) => {
+    console.log(node)
+    const doc = node.childrenDocumentationJs
+      .filter(def => def.kind !== `typedef` && def.memberof)
+      .map(def => {
+        if (!context.apicalls) {
+          // When an api call list is not available, the line numbers from jsdoc
+          // might be useful. Just for actions.mdx right now.
+          def.codeLocation.file = node.relativePath
+          if (!def.codeLocation.file) {
+            def.codeLocation = null
+          }
+        } else {
+          // API pages having apicalls did not query for this in the page query,
+          // so just remove it instead. Having one that returns nothing supresses
+          // documentation git links completely.
+          def.codeLocation = null
+        }
+        return def
+      })
+    return acc.concat(doc)
+  }, [])
+
+  let funcs = sortBy(docs, func => func.name)
+
+  const mergedFuncs = funcs.map(func => {
+    return {
+      ...func,
+      ...normalized.find(n => n.name === func.name),
+    }
+  })
+
+  return mergedFuncs
+}
+
+/*
+  return (
+    <React.Fragment>
+      <h2>APIs</h2>
+      <ul>
+        {funcs.map(node => (
+          <li key={`function list ${node.name}`}>
+            <a href={`#${node.name}`}>{node.name}</a>
+          </li>
+        ))}
+      </ul>
+      <h2>Reference</h2>
+      <APIReference docs={mergedFuncs} />
+    </React.Fragment>
+  )
+
+*/
```

Comment:
This can be removed?

Reply:
Yes, definitely. I should have put a TODO on there.